drizzlepac/mapreg.py

```diff
@@ -51,6 +62,226 @@ def wcs_pix2sky(self, *args, **kwargs):
+    coordinates, DS9 will use the WCS info from the image onto which the region file
+    was loaded. The :py:func:`~drizzlepac.mapreg.MapReg` task tries to automate this process.
+
+    **NOTE 2:** :py:func:`~drizzlepac.mapreg.MapReg` relies on the
+    `stregion <https://github.com/spacetelescope/stregion>`_ package for region file
+    parsing and coordinate transformation. Unfortunately, as of writing,
+    **stregion** does not consider distortion corrections when performing coordinate
+    transformations. Therefore, there might be a slight discrepancy between the
+    regions produced by :py:func:`~drizzlepac.mapreg.MapReg` and the DS9 regions obtained as described in
+    the NOTE 1 above.
+
+    **NOTE 3:** :py:func:`~drizzlepac.mapreg.MapReg` does not take into account pointing errors and thus the
+    produced region files can be somewhat misaligned compared to their intended
+    position around the sources identified in the "reference" image. Threfore, it is
+    highly recommended that the produced region files be loaded into DS9 and their
+    position be adjusted manually to include the sources of interest (or to avoid
+    the regions that need to be avoided).
+    If possible, the *include* or *exclude* regions should be large enough as to
+    allow for most pointing errors.
+
+    Examples
+    --------
+    Let's say that based on some image ``refimg.fits`` we have produced a "master"
+    reference image (``master.reg``) that includes regions around sources that we want
+    to use for image alignment in task :py:func:`~drizzlepac.tweakreg.TweakReg` and excludes regions that we
+    want to avoid being used for image alignment (e.g, diffraction spikes, saturated
+    quasars, stars, etc.). We save the file ``master.reg`` in sky CS (e.g., ``fk5``).
+
+    Also, let's assume that we have a set of images ``img1.fits``, ``img2.fits``, etc.
+    with four FITS extensions named 'SCI' and 'DQ'. For some of the extensions,
+    after analizing the ``img*.fits`` images we have identified parts of the chips that
+    cannot be used for image alighnment. We create region files for those extensions
+    and save the files in image CS as, e.g., ``img1_chip_sci2.reg`` (in our example
+    this will be the only chip that needs "special" treatment).
+
+    Finally, let's say we want to "replicate" the "master" region file to all SCI
+    exctensions of the ``img*.fits`` images as well as to the 2nd DQ extension and to
+    the 8th extension of the ``img*.fits`` images.
+
+    To do this we run:
+
+        >>> mapreg(input_reg = 'master.reg', images='img*.fits',
+        ...        img_wcs_ext='sci;dq,2;8', chip_reg='None,
+        ...        img1_chip_sci2.reg,None,None,None,None')
+
+    This will produce six region files in the ./regions subdirectory for *each*
+    input image::
+
+        ``img1_sci1_twreg.reg``,    ``img1_sci2_twreg.reg``,    ``img1_sci3_twreg.reg``,
```

Comment:
You don't need double backticks if the text is already in a literal block (`::`). Looks weird when you use both, please see https://drizzlepac--2078.org.readthedocs.build/en/2078/api/drizzlepac.mapreg.MapReg.html#drizzlepac.mapreg.MapReg